src/modules/party-external-rating/party-external-rating.ts

```diff
@@ -0,0 +1,17 @@
+export interface PartyExternalRating {
```

Comment:
can we call the file: `external-rating.interface.ts`? that way we know from the file name the purpose of it

Reply:
Sure - do we want to use `party-external-rating.interface.ts` (rename the interface to `ExternalRating` instead)?

Reply:
`party-external-rating.interface.ts` as the file name
and `PartyExternalRating` as the name of the interface above